src/Log/Listener/ConsoleOutputListener.php

```diff
@@ -11,7 +11,7 @@ class ConsoleOutputListener implements Listener {
      */
     protected $consoleOutput;
 
-    protected $outputPadLength = 80;
+    protected $outputPadLength = 0;
```

Comment:
@frickelbruder not sure why ``$outputPadMinLength`` was there in the first place (unused)